bundle/compliance/cis_k8s/rules/cis_4_2_4/data.yaml

```diff
@@ -48,4 +48,5 @@ metadata:
   - Kubelet
   benchmark:
     name: CIS Kubernetes V1.23
-    version: v1.0.0
\ No newline at end of file
+    version: v1.0.0
+    id: cis_k8s
```

Comment:
in some files the last line is at `version` and at others at a newline after `version` (EOF) so we get a "no new line at end of file" 

this makes some files have a diff of just `id` and some of `version` and `id` 

we might want to enforce insertion of missing new lines in a pre-commit hook (or similar?)

Reply:
@oren-zohar ^